dockers/docker-dhcp-relay/isc-dhcp-relay.j2

```diff
@@ -0,0 +1,7 @@
+SERVERS="{{ dhcp_servers | join(' ') }}"
+
+INTERFACES="{{ minigraph_vlan_interfaces | join(' ') }}"
```

Comment:
ok, since we are in the process to build one image (an image that can be deployed to the field) without any controller by the mgmt repo. 

but I agree with you to let this outside the docker, so we can do this in the base image to decide whether to start this docker or not.